bach/bach/series/series_numeric.py

```diff
@@ -49,6 +49,15 @@ def round(self, decimals: int = 0) -> 'SeriesAbstractNumeric':
             expression=Expression.construct(f'round(cast({{}} as numeric), {decimals})', self)
         )
 
+    def cut(self, bins: int, right: bool = True) -> 'SeriesAbstractNumeric':
+        """
+        Segments values into bins.
+        :param bins: The amount of bins to segment data into
```

Comment:
```suggestion
        Segments values into bins.

        :param bins: The amount of bins to segment data into
```

without the space, this doesn't render nicely in html.

To reproduce:
```bash
cd bach/docs  # this assumes the virtual environment is setup for sphinx
make html
xdg-open bach/docs/build/html/Series/bach.SeriesAbstractNumeric.cut.html
```